packages/transport/src/modules/relationships/local/PeerDeletionInfo.ts

```diff
@@ -0,0 +1,27 @@
+import { serialize, validate } from "@js-soft/ts-serval";
+import { CoreSerializable, ICoreSerializable } from "../../../core";
+
+export enum PeerDeletionStatus {
+    ToBeDeleted = "ToBeDeleted",
+    Deleted = "Deleted"
+}
+
+export interface PeerDeletionInfoJSON {
+    deletionStatus: PeerDeletionStatus;
+}
```

Comment:
Why was the deletionDate removed everywhere again? The App wants to display this information, doesn't it?

Reply:
bkb doesn't support it yet.

Reply:
So we'll add it later in a separate PR?

Reply:
check.